### ISSUE TITLE
Contract should sub royalty price ahead of computing 2% platform tax.

### ISSUE THREAD

**Issue by Erreth-Akbe** [SparkNFT/SparkLink#90]
## Summary
Contract should sub royalty price ahead of computing 2% platform tax.
## Should finish before contract deployed on Polygon

**Comment by SantanaSP** [comment]
Please follow one of our issue templates when you open a new issue. I don’t know what the issue is since you didn't describe it well. Reopen this when you rewrite it.
